packages/ftk-imager.vm/ftk-imager.vm.nuspec

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>ftk-imager.vm</id>
+    <version>4.7.1.20231207</version>
```

Comment:
[nitpick] fourth segment is not needed

Reply:
I though the version was 4.7.1 as that is what is in the url. But you are right the tool version is 4.7.1.2, so this version is correct, sorry for the confusion.